src/components/cqlLibraryList/CqlLibraryList.tsx

```diff
@@ -296,6 +304,92 @@ export default function CqlLibraryList({
     setAnchorEl(null);
   };
 
+  const columnsToBeAdded = [
+    {
+      header: "Name",
+      accessorKey: "cqlLibraryName",
+      cell: (info) => (
+        <button
+          type="button"
+          onClick={() =>
+            navigate(`/cql-libraries/${info.row.original.id}/edit/details`)
```

Comment:
Is this navigation a new requirement ?

Reply:
I'm not sure. This was just moved, so it must've been from an older ticket